File: core/Dtos/DeploymentDto.cs
namespace core.Dtos
{
    public class DeploymentDto
    {
        public int Id { get; set; }
        public int CVRefId { get; set; }
        public DateTime TransactionDate { get; set; }
        public int Sequence {get; set;}
        //public string StageName { get; set; }
        //public string NextStageName { get; set; }
        public int NextSequence {get; set;}
        public DateTime NextStageDate { get; set; }
    }
}

File: client2/src/app/shared/models/process/deploymentdto.ts
export interface IDeploymentDto
{
	id: number;
	deployCVRefId: number;
	transactionDate: Date;
	sequence: number;
	//stageName: string;
	nextSequence: number;
	//nextStageName: string;
	nextStageDate: Date;
}

export class DeploymentDto implements IDeploymentDto
{
	id=0;
	deployCVRefId = 0;
	transactionDate = new Date();
	sequence = 0;
	//stageName = '';
	nextSequence = 0;
	//nextStageName = '';
	nextStageDate = new Date();
}